recipes/openmesh/all/conandata.yml

```diff
@@ -1,11 +1,16 @@
 sources:
+  "10.0.0":
```

Comment:
Openmesh only uses `major.minor` versioning

Reply:
Thanks! Fixed it.